plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/TypePattern.java

```diff
@@ -29,6 +30,8 @@
 
 public class TypePattern
 {
+    private static final String ANY_TYPE = "any";
```

Comment:
this can be migrated to "type class" introduced in https://github.com/trinodb/trino/pull/11510/commits/3a394839bca3163ab04ade89c55abe367c4a9773

Reply:
It would be hard to define the "any" type class, especially that type classes don't support parameterized types today.

https://github.com/trinodb/trino/pull/11515 this should do

Reply:
done